libs/gen/Garnet.gen.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework> 
```

Comment:
the `TargetFrameworks` from the `Directory.Build.props` is being injected here which breaks this